client/assets/components/document/document_mail/document_mail.html

```diff
@@ -1,6 +1,6 @@
 <div class="detail vertical grid-content">
   <!-- show head field linked with a url. -->
-  <h3 class="head-field" ng-if="doc.subject"><a href="#"
+  <h3 class="head-field" ng-if="doc.subject"><a href="http://asfmail.lucidworks.io/"
```

Comment:
for mail docs, I don't think there is a fully qualified url as the ID, as the mail parsing step truncates to just the "%3CCANtJApz-c28BbFkGzFf6PxieYk_V_-==xJcc_5nO47h_XfPpwQ@mail.gmail.com%3E" at the end.  Easy enough to prefix that with "http://asfmail.lucidworks.io/mail_files/$project_name/$YYYYMM.mbox/" though!